### PR TITLE
Feat/elasticsearch snapshot module

### PR DESCRIPTION
**Pull request by jamatute**
Creation of module to do and delete snapshots of an Elasticsearch database

### CODE REVIEW

File: lib/ansible/modules/database/elasticsearch/elasticsearch_snapshot.py [hunk 1]
```diff
@@ -0,0 +1,238 @@
+#!/usr/bin/python3
```
Comment (alexppg) [inline lib/ansible/modules/database/elasticsearch/elasticsearch_snapshot.py]: Shouldn't be pointing to main python, probably with env?

File: lib/ansible/modules/database/elasticsearch/elasticsearch_snapshot.py [hunk 2]
```diff
@@ -0,0 +1,238 @@
...
+
+DOCUMENTATION = '''
+---
+module: elasticsearch_snapshot
+short_description: Do snapshots of elasticsearch indices
+description: Do snapshots of elasticsearch indices
+version_added: '2.5'
+requirements:
+  - "python >= 3.0"
+  - requests
```
Comment (alexppg) [inline lib/ansible/modules/database/elasticsearch/elasticsearch_snapshot.py]: Same as above, ansible doesn't officially support python3, as fas as I know.

File: lib/ansible/modules/database/elasticsearch/elasticsearch_snapshot.py [hunk 3]
```diff
@@ -0,0 +1,238 @@
...
+        description:
+          - Number of retries of failed http/https requests
+        default: 3
+        required: false
+    sleep:
+        description:
+          - Sleep between checks of the snapshot process
+        default: 15
+        required: false
+author: jamatute (@jamatute)
```
Comment (alexppg) [inline lib/ansible/modules/database/elasticsearch/elasticsearch_snapshot.py:67]: Maybe put email?

File: lib/ansible/modules/database/elasticsearch/elasticsearch_snapshot.py (hunk 1 above)
Comment (jamatute) [reply lib/ansible/modules/database/elasticsearch/elasticsearch_snapshot.py]: Fixed: f40b12868

**Comment by alexppg** [comment]
LGTM

File: lib/ansible/modules/database/elasticsearch/elasticsearch_snapshot.py (hunk 2 above)
Comment (jamatute) [reply lib/ansible/modules/database/elasticsearch/elasticsearch_snapshot.py]: Fixed in ee1c8669c 

Although they do [support python3 for modules](http://docs.ansible.com/ansible/latest/dev_guide/developing_python3.html)

File: lib/ansible/modules/database/elasticsearch/elasticsearch_snapshot.py (hunk 3 above)
Comment (jamatute) [reply lib/ansible/modules/database/elasticsearch/elasticsearch_snapshot.py:67]: I prefer to use my github account

File: lib/ansible/modules/database/elasticsearch/elasticsearch_snapshot.py (hunk 2 above)
Comment (alexppg) [reply lib/ansible/modules/database/elasticsearch/elasticsearch_snapshot.py]: Cool.

**Pull request merged**